README.md

```diff
@@ -44,6 +44,7 @@
 - install gems bundle i
 - setup database rails db:setup 
 - run rails server rails s.
+- rails c
 
 ### Prerequisites
 
```

Comment:
- [ ] The **MIT** license link in the **License** header  is not working. Broken links really hurt those checking your work and this makes the developer unprofessional. Kindly always double-check all your links before submitting your app.